src/main/java/com/example/teamtest/TeamTestApplication.java

```diff
@@ -9,5 +9,5 @@ public class TeamTestApplication {
 	public static void main(String[] args) {
 		SpringApplication.run(TeamTestApplication.class, args);
 	}
-
+//1
 }
```

Comment:
```suggestion
// 2
}
```

@HyonHyonKOR 이거라니까요!!!!

Reply:
해결